src/mesh/voxel_cloud.rs

```diff
@@ -283,8 +283,8 @@ impl VoxelCloud {
                 //left
                 plane: Plane::new(
                     &Point3::origin(),
-                    &Vector3::new(0.0, 1.0, 0.0),
-                    &Vector3::new(0.0, 0.0, -1.0),
+                    &Vector3::new(0.0, -1.0, 0.0),
```

Comment:
So this before produced a normal facing in the other direction?

Reply:
yes, there were several mixed-up planes. the results now look ok and there is also a test